aries-site/src/pages/components/header.mdx

```diff
@@ -16,7 +16,7 @@ Header is extremely versatile and can be built by mixing and matching a variety
   designer="https://designer.grommet.io/header?id=HPE-design-system-hpedesignsystem-hpe-com&mode=edit"
   docs="https://v2.grommet.io/header?theme=hpe"
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/header/HeaderExample.js"
-  figma="https://www.figma.com/file/krgk9GTsccqOvcTm8OVzH2/HPE-Header-Component?node-id=0%3A2"
+  figma="https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?type=design&node-id=3028-58404&t=KjhQZHbAZR51Nmwu-11"
```

Comment:
@ashifalinadaf I notice this is linking to Global Header, but this page was initially created for the more generic "Header" component. I'm wondering if this is the right link or not.